### PR TITLE
update ray version in default template and code references

### PR DESCRIPTION
**Pull request by KPostOffice**
## Checks
- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [x] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: tests/unit_test.py
Comment (Maxusmusti) [inline tests/unit_test.py:2299]: Assuming you want to uncomment these?

**Review by Maxusmusti — CHANGES_REQUESTED** [review]

File: tests/unit_test.py
Comment (KPostOffice) [reply tests/unit_test.py:2299]: Oops, yes thanks

**Review by Maxusmusti — COMMENTED** [review]
/lgtm

**Review by anishasthana — COMMENTED** [review]
/lgtm

File: custom-nb-image/Dockerfile
Comment (dimakis) [inline custom-nb-image/Dockerfile:16]: nit:
can we update/  change this to leave out the versions? I think its pretty self explanatory anyway

**Review by dimakis — COMMENTED** [review]
lgtm

File: custom-nb-image/Dockerfile
Comment (Maxusmusti) [reply custom-nb-image/Dockerfile:16]: I'll wait until after that comment is removed/updated to approve, @KPostOffice

**Review by Maxusmusti — APPROVED** [review]
LGTM

**Pull request merged**
